help/keyboard-shortcuts.md

```diff
@@ -141,6 +141,10 @@ in the Zulip app to add more to your repertoire as needed.
 * **Insert link**: `[Zulip website](https://zulip.org)` or <kbd>Ctrl</kbd> +
   <kbd>Shift</kbd> + <kbd>L</kbd>
 
+* **Insert code formatting**: `` `code` `` or ```` ```code``` ```` or <kbd>Ctrl</kbd> +
+  <kbd>Shift</kbd> + <kbd>C</kbd>, depending on your text selection.
+  See [contextually appropriate code formatting](/help/code-blocks#insert-code-formatting).
```

Comment:
Let's drop ", depending on your text selection"; I don't think it's clear what it means here, and the link will clarify the details.